R/equals_na_linter.R

```diff
@@ -1,6 +1,7 @@
 #' Equality check with NA linter
 #'
-#' Check for `x == NA` and `x != NA`
+#' Check for `x == NA` and `x != NA`. One should instead use `is.na()` function
```

Comment:
Not a fan of (IMO) clumsy wording like "one should", how about

Such usage is almost surely incorrect -- checks for missing values should be done with `[is.na()]`.

Reply:
Thanks. Changed to the suggested phrasing.